apps/web/app/dashboard/admin/page.tsx

```diff
@@ -158,6 +181,10 @@ function ServerStatsSection() {
             <TableCell className="lg:w-2/3">Pending Crawling Jobs</TableCell>
             <TableCell>{serverStats.pendingCrawls}</TableCell>
           </TableRow>
+          <TableRow>
+            <TableCell className="lg:w-2/3">Failed Crawling Jobs</TableCell>
+            <TableCell>{serverStats.failedCrawls}</TableCell>
+          </TableRow>
```

Comment:
How about we change the table to be a bit more 2d?

As in:

| | Pending | Failed |
-----|------|-----|
|Crawling jobs | 0 | 0 |
|Inference jobs | 0 | 0 |
|Search jobs | 0 | - |

Reply:
For now the indexing and openai are going to be based off the queue status.